odbc-api/tests/integration.rs

```diff
@@ -2129,3 +2129,33 @@ fn many_diagnostic_messages() {
 
     // We do not have an explicit assertion, we are just happy if no integer addition overflows.
 }
+
+/// Ensures access to driver and data source info is synchronized correctly when multiple threads
+/// attempt to query it at the same time. First, we query the list of the known drivers and data
+/// sources on the main thread. Then we spawn multiple threads that attempt to query these lists in
+/// parallel. Finally we compare the lists to ensure they match the list we found on the main
+/// thread.
+#[test_case(MSSQL; "Microsoft SQL Server")]
```

Comment:
A normal #[test] is actually enough, as we never instantiate any Connection in this test. The purpose of these lines is to call the test function with the associated `Profile`. Since the argument is ignored anyway we can do without this mechanism in this test.

Reply:
Ah ok sounds good. I thought they were making the drivers available but I see now that it's only used for `Connection`